### PR TITLE
Added Top button

### LINKED ISSUE #63: Top button for the home and explore page.
**Is your feature request related to a problem? Please describe.**
The options to sign up, logout, explore or other such options are available on the top of the home page and explore page, but when you are scrolling down and you want to use one of these options you have to scroll all the way to the top of the page Which is tidious.

**Describe the solution you'd like**
A very good solution to this problem will be a top button on the bottom right corners of the home and explore page. This button can have an upward arrow or top written on it and it will take you to the top of the page when clicked. The button can be customised and made attractive using CSS.

**Are you willing to contribute to this issue?** 
Yes, I am a participant in GsSOC'21.

**Describe alternatives you've considered**
I thought of using a sticky navigation bar but personally I don't like that kind of navigation bar, it covers the contents and is irritating.

### PR DESCRIPTION
**Pull request by kanishka8276**
## Description
- Added top button to all the pages by just changing layout.html and layout.css. This button will take you to the top of the page. And the scrolling is smooth.
- #63 

## Affected Dependencies
NA

## How has this been tested?
Ran it in my localhost

- [x] I have followed the [Contribution Guidelines](https://github.com/Praful932/Kitabe/blob/master/CONTRIBUTING.md).
- [x] My changes do not edit/add any unrequired files.
- [x] My changes are covered by tests.

Screenshot (63)

When on the top of the page the button is not displayed.
Screenshot (64)
Screenshot (65)
On hover
Screenshot (66)

### CODE REVIEW

File: templates/mainapp/layout.html [hunk 1]
```diff
@@ -136,12 +137,23 @@ <h4 class="modal-title" id="booktitle">Basic Modal</h4>
         </div>
     </div>
     </div>
-
+	<script>
```
Comment (maneprajakta) [inline templates/mainapp/layout.html:140]: add this script [here](https://github.com/Praful932/Kitabe/blob/61f4fbf6d7e2366b11a563d17603d6098b3f5807/templates/mainapp/layout.html#L168) we already have a script tag right, so all js should be inside it no need to add another block.

**Review by maneprajakta — CHANGES_REQUESTED** [review]
Now you got an idea of how template inheritance works :rocket:.

File: templates/mainapp/layout.html (hunk 1 above)
Comment (kanishka8276) [reply templates/mainapp/layout.html:140]: Ok sure!

**Comment by kanishka8276** [comment]
@maneprajakta  done!

File: templates/mainapp/layout.html [hunk 2]
```diff
@@ -137,15 +137,7 @@ <h4 class="modal-title" id="booktitle">Basic Modal</h4>
...
-    if (document.body.scrollTop > 0 || document.documentElement.scrollTop > 0) {
-        document.getElementById("myBtn").style.display = "block";
-    } else {
-        document.getElementById("myBtn").style.display = "none";
-    }
-}) 
-	</script>
+	
 	<a href='#top'>
 	<button  id="myBtn" title="Go to top"><i class="fa fa-angle-up"></i></button></a>
```
Comment (maneprajakta) [inline templates/mainapp/layout.html:142]: If you could change the button id from myBtn to topBtn seems more accurate.

**Review by maneprajakta — COMMENTED** [review]
Rest all seems fine.

**Comment by kanishka8276** [comment]
@maneprajakta  done!

**Comment by Praful932** [comment]
@kanishka8276 would suggest to keep the button size smaller and the button style consistent with project theme

**Comment by kanishka8276** [comment]
Ok I will make it smaller. Can you tell me what type of changes to be made in the styling of the button.

**Comment by kanishka8276** [comment]
Screenshot (67)
Screenshot (68)

**Comment by maneprajakta** [comment]
@kanishka8276 we have deployed your PR you can visit it [here](https://kitabe-pipeline-pr-88.herokuapp.com/).
In mobile view 
-  The button covers a part of the footer content :
WhatsApp Image 2021-03-14 at 6 25 05 PM
- Also the there is no hover in mobile so keep the same color black for the small screen even after a click.

**Comment by kanishka8276** [comment]
Ok!

**Comment by kanishka8276** [comment]
@Praful932 @maneprajakta Done!

**Review by Praful932 — APPROVED** [review]

**Comment by maneprajakta** [comment]
@kanishka8276 reduce right padding a bit. It will work. For small screen.
WhatsApp Image 2021-03-14 at 10 03 07 PM

**Review by maneprajakta — APPROVED** [review]

**Pull request merged**